testnet/stacks-node/Stacks.toml

```diff
@@ -23,7 +23,7 @@ rpc_bind = "127.0.0.1:20443"
 # password = "secret"
 # timeout = 30
 # local_mining_public_key = "04ee0b1602eb18fef7986887a7e8769a30c9df981d33c8380d255edef003abdcd243a0eb74afdf6740e6c423e62aec631519a24cf5b1d62bf8a3e06ddc695dcb77"
-# burnchain_op_tx_fee = 1000
+# satoshis_per_block_commit_op = 1000
```

Comment:
Event though this is commented out, I think this should probably be `= 5500`

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2317/commits/46c89a2f299a5593d5137fd25a8a210f6035dd1c